recipes/pwkit/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "pwkit" %}
+{% set version = "0.8.9" %}
+{% set sha256 = "ee706f8b1675ea5be04b4ec2052a596d5ade3819b26d876b4af94373072110c5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=files.txt
+  skip: true  # [win]
+  entry_points:
+    - astrotool = pwkit.cli.astrotool:commandline
+    - casatask = pwkit.environments.casa.tasks:commandline
+    - imtool = pwkit.cli.imtool:commandline
+    - pkcasascript = pwkit.environments.casa.scripting:commandline
+    - pkenvtool = pwkit.environments:commandline
+    - wrapout = pwkit.cli.wrapout:commandline
+
+requirements:
+  build:
+    - numpy
+    - pathlib  # [py2k]
+    - python
+    - setuptools
+    - six
+
+  run:
+    - numpy
+    - pathlib  # [py2k]
+    - python
+    - six
+
+test:
```

Comment:
Can you add tests for the `entry_points`:

https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#test-commands

Reply:
Done, skipping the CASA-related ones, which require a very large software stack to be installed in order to run.